rel/files/mongooseim.cfg

```diff
@@ -77,13 +77,17 @@
 %%
 %% loglevel: Verbosity of log files generated by ejabberd.
 %% 0: No ejabberd log at all (not recommended)
-%% 1: Critical
-%% 2: Error
-%% 3: Warning
-%% 4: Info
-%% 5: Debug
-%%
-{loglevel, 3}.
+%%% -1: none;
```

Comment:
`none` means no loglevel at all, `all` is the one that accepts all of them, so this is the correct order. The numbers, it's mostly because we already had that functionality, and because the syslog standard recognises both numbers and keywords. I do think that pretty much keywords (atoms) are better, even the OTP team took that decision for logger, but well, just to keep the same functionality than before, albeit extended. We can discuss later on in the team what do we want to do here 🙂

Reply:
I think I misunderstood the `all` and `none` in this place. You are completely right.